hypothesis-python/docs/index.rst

```diff
@@ -57,27 +57,34 @@ check out some of the
 .. toctree::
   :maxdepth: 1
   :hidden:
+  :caption: Hypothesis
 
   quickstart
   details
-  settings
   data
-  extras
+  settings
+  database
+  stateful
+  reproducing
   ghostwriter
+  examples
+  extras
   django
   numpy
```

Comment:
"hypothesis for data science" is maybe more concise than "hypothesis for the scientific stack" here?

Reply:
made this change 👍